projects/angular/src/button/API.md

```diff
@@ -0,0 +1,160 @@
+| --clr-btn-link-hover-bg-color                   | Changes the link button `:hover` background color.                                      |
+| --clr-btn-link-disabled-color                   | Changes the link button `disabled` text color.                                          |
+| --clr-btn-link-disabled-bg-color                | Changes the link button `disabled` background color.                                    |
+| --clr-btn-link-disabled-border-color            | Changes the link button `disabled` border color.                                        |
+| --clr-btn-link-checked-color                    | Changes the checked checkbox link button text color.                                    |
+| --clr-btn-link-checked-bg-color                 | Changes the checked checkbox link button background color.                              |
+| --clr-btn-inverse-color                         | Changes the inverse button text color.                                                  |
+| --clr-btn-inverse-bg-color                      | Changes the inverse button background color.                                            |
+| --clr-btn-inverse-border-color                  | Changes the inverse button border color.                                                |
+| --clr-btn-inverse-hover-color                   | Changes the inverse button `:hover` text color.                                         |
+| --clr-btn-inverse-hover-bg-color                | Changes the inverse button `:hover` background color.                                   |
+| --clr-btn-inverse-box-shadow-color              | Changes the inverse button `:active` box shadow.                                        |
+| --clr-btn-inverse-disabled-color                | Changes the inverse button `disabled` text color.                                       |
+| --clr-btn-inverse-disabled-bg-color             | Changes the inverse button `disabled` background color.                                 |
+| --clr-btn-inverse-disabled-border-color         | Changes the inverse button `disabled` border color.                                     |
+| --clr-btn-inverse-checked-color                 | Changes the checked checkbox inverse button text color.                                 |
+| --clr-btn-inverse-checked-bg-color              | Changes the checked checkbox inverse button background color.                           |
+| --clr-btn-icon-disabled-color                   | Changes the icon button `disabled` text color.                                          |
+| --clr-btn-group-focus-outline                   | Changes the `:focus` outline color of a checkbox or radio inside a button group button. |
+
+## Class names
+
+| Class                    | Description               |
+| ------------------------ | ------------------------- |
+| .btn                     | button                    |
+| .btn.btn-info            | info button               |
```

Comment:
Can you change these to be class names instead of selectors? 

The description should be in a similar format to the above table.

Reply:
Just the class name is not good documentation. The classes must be used in a specific structure.

Reply:
It's important that these remain consistent with other components, and listing the selectors in this way will lead to longer and longer class names which will be complicated for users to parse/read and even more difficult to maintain.